vaadin-spring-tests/test-ts-services/src/main/java/com/vaadin/flow/connect/PackagePrivateEndpoint.java

```diff
@@ -0,0 +1,23 @@
+package com.vaadin.flow.connect;
+
+import javax.annotation.Nullable;
+
+import com.vaadin.flow.server.connect.Endpoint;
+import com.vaadin.flow.server.connect.auth.AnonymousAllowed;
+
+/**
+ * Simple Vaadin Connect Service definition.
+ */
+@Endpoint
+public class PackagePrivateEndpoint {
```

Comment:
Should this class be package-private then?

Reply:
ah, shame on me, fixed